common/utils/HttpClient.js

```diff
@@ -182,8 +182,8 @@ class HttpClient {
           try {
             const errResponse = JSON.parse(body);
             err.error = errResponse;
-          } catch (err) {
-            logger.info('Error occurred while parsing http response- ', err.error);
+          } catch (parseErr) {
```

Comment:
`parseErr` ?

Reply:
There are two err here . `err` is defined in line 141. parseErr can occur in 183

Reply:
This is required since metering responds with error message in `body`. Hence the parsing will fail in line 183

Reply:
ok.